shell/detail/workload/index.vue

```diff
@@ -393,7 +381,7 @@ export default {
         :weight="4"
       >
         <ResourceTable
-          v-if="value.pods"
+          v-if="value.pods?.length"
```

Comment:
I think we should either display a message that there aren't any pods or we should hide the tab. Seems like we could end up with a blank tab.

Reply:
I've updated. The tab now only shows if the workload type supports them (i.e. has a pod matchExpression) and now we always show the table. This was lo-bar, ideally for both job and pod based workloads we should probably warn if nothing matches (rather than show an empty list)